staff/roberto-perez/blog/post.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Lorem ipsum dolor sit amet</title>
+</head>
+<body>
+    <header>
+        <h1>Blog name</h1>
+        <h2>A basic semantic blog</h2>
+    </header>
+    <section>
```

Comment:
And...if you change <section> and use <main>?

"The <main> tag specifies the main content of a document.
The content inside the <main> element should be unique to the document"